lib_xua/src/core/main.xc

```diff
@@ -555,6 +558,7 @@ int main()
                          c_sof, epTypeTableOut, epTypeTableIn, usbSpeed, xudPwrCfg);
             }
 
+#if (NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0) || XUA_HID_ENABLED || defined(MIDI)
```

Comment:
Do we want the buffer if only XUA_HID_ENABLED is enabled?

Reply:
HID handling is a little complicated now due to the changes added to support custom hid endpoints - as required for lib_awe). XUA_HID_ENABLED switches on/off the internal HID handling e.g. playback controls - this runs through XUA_Buffer_Ep(). Though im not sure it makes sense to have a device with this enabled